I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 965, in script
    " Je vais essayer de me détendre et de me coucher tôt pour réattaquer du bon pied demain. Après la journée que j’ai passé, je n’ai pas le courage de m‘en occuper ce soir":
  File "game/script.rpy", line 972, in <module>
    jump _2_6E
NameError: name 'flag_2_3B_1' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 965, in script
    " Je vais essayer de me détendre et de me coucher tôt pour réattaquer du bon pied demain. Après la journée que j’ai passé, je n’ai pas le courage de m‘en occuper ce soir":
  File "C:\Users\match\Documents\SHS_JeuVideo\SHS_Cybersecurity_game\renpy-7.3.5-sdk\renpy\ast.py", line 1628, in execute
    choice = renpy.exports.menu(choices, self.set, args, kwargs, item_arguments)
  File "C:\Users\match\Documents\SHS_JeuVideo\SHS_Cybersecurity_game\renpy-7.3.5-sdk\renpy\exports.py", line 956, in menu
    condition = renpy.python.py_eval(condition)
  File "C:\Users\match\Documents\SHS_JeuVideo\SHS_Cybersecurity_game\renpy-7.3.5-sdk\renpy\python.py", line 2059, in py_eval
    return py_eval_bytecode(code, globals, locals)
  File "C:\Users\match\Documents\SHS_JeuVideo\SHS_Cybersecurity_game\renpy-7.3.5-sdk\renpy\python.py", line 2052, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/script.rpy", line 972, in <module>
    jump _2_6E
NameError: name 'flag_2_3B_1' is not defined

Windows-8-6.2.9200
Ren'Py 7.3.5.606
Please do not hack me 1.0
Tue May 12 21:13:09 2020
